File: eBarberShop/UI/ebarbershop_mobile/lib/models/narudzbe/narudzbe.dart
import 'package:json_annotation/json_annotation.dart';

part 'narudzbe.g.dart';

@JsonSerializable()
class Narudzbe {
  int narudzbeId;
  String brojNarudzbe;
  DateTime datumNarudzbe;
  double ukupanIznos;
  bool status;
  bool? otkazano;
  int korisnikId;

  Narudzbe(this.narudzbeId, this.brojNarudzbe, this.datumNarudzbe,
      this.ukupanIznos, this.status, this.otkazano, this.korisnikId);

  factory Narudzbe.fromJson(Map<String, dynamic> json) =>
      _$NarudzbeFromJson(json);

  Map<String, dynamic> toJson() => _$NarudzbeToJson(this);
}

File: eBarberShop/eBarberShop.Services/Database/Narudzbe.cs
namespace eBarberShop.Services.Database
{
    public partial class Narudzbe
    {
        public int NarudzbeId { get; set; }
        public string BrojNarudzbe { get; set; }
        public DateTime DatumNarudzbe { get; set; }
        public decimal UkupanIznos { get; set; }
        public bool Status { get; set; }
        public bool? Otkazano { get; set; }
        public int KorisnikId { get; set; }

        public virtual Korisnici Korisnik { get; set; }

        public virtual ICollection<NarudzbeDetalji> NarudzbeDetalji { get; set; } = new List<NarudzbeDetalji>();
        public virtual ICollection<PaymentDetail> PaymentDetail { get; set; } = new List<PaymentDetail>();

    }
}
